applications/consdb/Chart.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: consdb-hinfo
+version: 1.0.0
+description: Consolidated Database of Image Metadata
+type: application
+appVersion: dm-37511
+home: consdb.lsst.io
```

```suggestion
home: consdb.lsst.io
```

Delete non-existant homepage.